legacy_ui/dashboard/components/agent_status.py

```
@@ -0,0 +1,156 @@
+import json
+from pathlib import Path
+from datetime import datetime
+from typing import Dict, List, Optional, Tuple
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)